### ISSUE TITLE
make TaskWorker start/stop/install etc. available after deployment

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#6012]
with ref to #5943 
need to modify
https://github.com/dmwm/CRABServer/blob/master/setup.py

e.g. to put those files in 
```
TaskManager/<release>/slc7_amd64_gcc630/cms/crabtaskworker/<release>/bin/
```
or bettet yer in `TaskManager/<release>/bin` so that they'll be available as `current/bin`

**Comment by belforte** [comment]
apparently `setup.py` can put scripts in `crabtaskworker/<version>/bin`
https://docs.python.org/2/distutils/apiref.html?highlight=setup#distutils.core.setup
https://github.com/dmwm/CRABServer/blob/04c65aa711e375e2938e253e4822f949d9ed296f/setup.py#L228
but it will be a single `bin` directory for TaskWorker and Publisher, since we have a single project.
So I may have to avoid the subdirectories and rather rename the files.
I need to do something crude and simple, avoid going down the route of learning all details of python distutils.

**Comment by belforte** [comment]
Nope. When I add
```
     scripts=['scripts/Deployment/TaskWorker/%s' % x for x in ['start.sh', 'stop.sh']],
```
to `setup.py`, it puts them in `lib/python2.7/site-packages/`

**Comment by belforte** [comment]
OK. I believe that the thing is:
**the build/deploy procedure, i.e. `setup.py`, will reproduce in `TaskManager/current/../data/script` the tree structure in git `src/script`**
which I think I figured out before but forgot.
Will try to move files around.

**Comment by belforte** [comment]
closed via #6013
